kingpin/actors/rightscale/test/integration_mci_and_st.py

```diff
@@ -0,0 +1,138 @@
+                     'images': self.images},
+            dry=True)
+        yield actor.execute()
+
+    @attr('rightscale', 'integration')
+    @testing.gen_test(timeout=60)
+    def integration_01b_create_mci(self):
+        actor = mci.MCI(
+            options={'name': self.name,
+                     'description': self.name,
+                     'state': 'present',
+                     'images': self.images})
+        yield actor.execute()
+
+    @attr('rightscale', 'integration')
+    @testing.gen_test(timeout=60)
+    def integration_03a_update_mci(self):
+        self.images[0]['instance_type'] = 'm1.large'
+        self.images[1]['instance_type'] = 'm1.large'
+
+        actor = mci.MCI(
+            options={'name': self.name,
+                     'description': self.name,
+                     'state': 'present',
+                     'commit': 'Changed instance type to m1.large',
+                     'images': self.images})
+
+        yield actor.execute()
+
+    @attr('rightscale', 'integration')
+    @testing.gen_test(timeout=60)
+    def integration_03b_update_mci(self):
+
+        actor = mci.MCI(
+            options={'name': self.name,
+                     'description': self.name,
+                     'state': 'present',
+                     'tags': 'some_new_tag',
+                     'commit': 'Added a tag',
+                     'images': self.images})
+
+        yield actor.execute()
+
+    @attr('rightscale', 'integration', 'dry')
+    @testing.gen_test(timeout=60)
+    def integration_05a_create_st_dry(self):
+        actor = server_template.ServerTemplate(
+            options=self.template,
+            dry=True)
+        yield actor.execute()
+
+    @attr('rightscale', 'integration')
+    @testing.gen_test(timeout=60)
+    def integration_05b_create_st(self):
+        actor = server_template.ServerTemplate(
+            options=self.template)
+        yield actor.execute()
+
+    @attr('rightscale', 'integration')
+    @testing.gen_test(timeout=60)
+    def integration_06a_delete_st(self):
+        self.template['state'] = 'absent'
+        actor = server_template.ServerTemplate(
+            options=self.template)
+        yield actor.execute()
+
+    @attr('rightscale', 'integration', 'dry')
+    @testing.gen_test(timeout=60)
+    def integration_09a_destroy_mci_dry(self):
+        actor = mci.MCI(
+            options={'name': self.name,
+                     'description': self.name,
+                     'state': 'absent',
+                     'images': self.images},
+            dry=True)
+        yield actor.execute()
+
+    @attr('rightscale', 'integration')
+    @testing.gen_test(timeout=60)
+    def integration_09b_destroy_mci(self):
+        actor = mci.MCI(
+            options={'name': self.name,
+                     'description': self.name,
+                     'state': 'absent',
+                     'images': self.images})
+        yield actor.execute()
```

Comment:
None of the integration tests have any asserts. Which means if the code is syntactically correct but doesn't actually do what it should - this wouldn't catch it. For work like this the integration tests are the only thing that matters. Unit tests have zero value here